processes/github_access.md

```diff
@@ -20,8 +20,13 @@ All projects at Defra must be created under one of the organisations within the
 - [Defra](https://github.com/DEFRA) for software
 - [Defra Data Science Centre of Excellence](https://github.com/-Defra-Data-Science-Centre-of-Excellence) for data science
 - [Defra design team](https://github.com/defra-design) for prototype designs
+- [aphascience](https://github.com/aphascience) for the APHA Scientific Computing Environment
 
-The organisation owners can be contacted to add you to their organisation.
+The organisation owners can be contacted to add you to their organisation:
+
+- for software development or design, use the #github-support channel on the Defra Digital Slack workspace
+- for data science visit the [Internal Defra DASH SharePoint site](https://defra.sharepoint.com/sites/Community448) for contact details
+- for the SCE visit the [Internal Defra SCE SharePoint site](https://defra.sharepoint.com/teams/Team741) for contact details
```

Comment:
As above, suggest something like `for scientific projects at APHA visit...`

Reply:
Changed